simplemma/lemmatizer.py

```diff
@@ -319,36 +320,19 @@ def text_lemmatizer(
 ) -> List[str]:
     """Convenience function to lemmatize a text using a simple tokenizer.
     Returns a list of tokens and lemmata."""
-    lemmata = []
-    last = "."  # beginning is initial
-    for match in simple_tokenizer(text, iterate=True):
```

Comment:
Here it used the iterator even though it's the list method.

Reply:
Within this function the iterator can stay.